# Generated from: /home/latty/link_dir/2015_li3ds/__ROS__/Ellipse_N/ROS/overlay_ws/src/sbg_ros_driver/sbgECom/src/binaryLogs/sbgEComBinaryLogEvent.h
# SbgLogEvent
uint32 timeStamp
uint16 status
uint16 timeOffset0
uint16 timeOffset1
uint16 timeOffset2
uint16 timeOffset3